copier.sublime-project

```diff
@@ -0,0 +1,28 @@
+{
+	"build_systems":
+	[
+		{
+			"file_regex": "^[ ]*File \"(...*?)\", line ([0-9]*)",
+			"name": "Anaconda Python Builder",
+			"selector": "source.python",
+			"shell_cmd": "\"/home/ben/Projects/copier/.venv/bin/python\" -u \"$file\""
+		}
+	],
+	"folders":
+	[
+		{
+			"path": "/home/ben/Projects/copier"
```

Comment:
Didn't you say you want no editor-specific files? 🤔

Imagine someone isn't called Ben! 😁😁

Reply:
Ouch. Yes that one slipped through.
I killed it once it entered `master`. :-|